### PR TITLE
Fix references in the manual.

### PR DESCRIPTION
**Pull request by jesnie**
Fix references in the manual.

A while ago we introduced a `Bibliography` section in our documentation.
I noticed that `notebooks/intro.md` hasn't been updated to use that, and thought I'd quickly fix it - and then it just turned into a whole *thing*.

So, I've

1. Converted `notebooks/intro.md` to `manual.rst`.
2. Had to move `GPflows.png`.
3. Updated the manual to integrate with our bibliography.

### CODE REVIEW

File: doc/sphinx/manual.rst [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+GPflow manual
+-------------
+
+You can use this document to get familiar with GPflow. We've split up the material into four different categories: basics, understanding, advanced needs, and tailored models. We have also provided a `flow diagram <_static/GPflows.png>`_ to guide you to the relevant parts of GPflow for your specific problem.
+
+
+GPflow 2
+--------
+
+Users of GPflow 1 should check the :doc:`notebooks/gpflow2_upgrade_guide`.
```
Comment (st--) [inline doc/sphinx/manual.rst:11]: Maybe time to retire this?

Comment (jesnie) [reply doc/sphinx/manual.rst:11]: > Maybe time to retire this?

Sure. But that's another thing, and a PR should only do one thing.

Comment (jesnie) [reply doc/sphinx/manual.rst:11]: BTW: #1823 tracks things I'd like to do for our documentation.

**Review by frgsimpson — APPROVED** [review]
LGTM!

**Pull request merged**